### ISSUE TITLE
Finalize Prod Login.gov Application by going through the their Approval/Review process

### ISSUE THREAD

**Issue by mdragon** [HHS/simpler-grants-gov#3256]
### Summary

Once our work is complete on #3133 we need to go through the Login.gov validation/approval process. Placeholder ticket for now to track that work 

### Acceptance criteria

- [ ] Submit Prod Application for Approval/Review
- [ ] ...
- [ ] Login.gov Prod Application is "Approved" and "Live"

**Comment by mdragon** [comment]
Created a duplicate of #3192 not realizing it existed.
